src/shared/src/logistics/redux/taskListEntityUtils.js

```diff
@@ -1,5 +1,19 @@
 import _ from 'lodash';
+import { createTempTaskList, replaceItemsWithItemIds } from './taskListUtils';
 
+function addTaskIdIfMissing(taskIds, taskId) {
+  const taskIdIndex = _.findIndex(taskIds, t => t === taskId);
+
+  if (taskIdIndex !== -1) {
+    return taskIds;
+  } else {
+    return taskIds.concat([taskId]);
```

Comment:
I dont understand this, if the task is already in taskIds, why do we add it again?

it will cause to assign it twice, as the validation is quite light on backend side ^^

Reply:
Nopes the `if (taskIdIndex !== -1) {` will ensure to add only if not already there and we use this list only in-memory from the app.